modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -283,6 +290,28 @@ private static String _getContentType(
 		return dtoConverter.getContentType();
 	}
 
+	private static Map<String, Object> _getDefaultVariables(
+		ObjectDefinition objectDefinition, Set<String> keys) {
+
+		List<ObjectField> objectFields =
+			ObjectFieldLocalServiceUtil.getObjectFields(
+				objectDefinition.getObjectDefinitionId());
+
+		Map<String, Object> defaultVariables = Collections.emptyMap();
```

Comment:
@p-albuquerque why are you using `Collections.emptyMap()` instead of `new HashMap<>()`?